addons/web/static/src/legacy/js/views/calendar/calendar_controller.js

```diff
@@ -486,6 +509,31 @@ var CalendarController = AbstractController.extend({
         const title = `${this.displayName} (${event.data.title})`;
         return this.updateControlPanel({ title });
     },
+
+    /**
+     * Update Attendee status in batch for recurrent events
+     * @private
+     * @param {OdooEvent} event
+     */
+     _onAttendeeStatus: async function(event) {
+         const self = this;
+         let recurrenceUpdate;
+         if (event.data.record.recurrency) {
+            recurrenceUpdate = await this._askRecurrenceUpdatePolicy();
+            event.data = _.extend({}, event.data, {
+                'recurrenceUpdate': recurrenceUpdate,
+            });
+         } else {
+            recurrenceUpdate = false;
```

Comment:
I would suggest that when you initialize `recurrenceUpdate` in line 520 you already assign it to false. Then you don't need this else block.